jws/jws.go

```diff
@@ -276,4 +265,4 @@ func (j Header) Encode() (string, error) {
 // Payload is a type to represent the [JWS Payload]
 //
 // [JWS Payload]: https://datatracker.ietf.org/doc/html/rfc7515#section-2
-type Payload any
+type Payload []byte
```

Comment:
Any benefit to introducing `Payload` as a type alias as opposed to just inlining `[]byte` as the type everywhere?

Reply:
great q @KendallWeihe was wondering the same actually. instinctively think we can get rid of it. want to toss up a subsequent PR to tackle?

Reply:
done https://github.com/TBD54566975/web5-go/pull/116